workflows/templates/finished-webhook-template.yaml

```diff
@@ -32,43 +32,67 @@ spec:
 
           init_logging()
 
-          metadata = {{inputs.parameters.THOTH_ADVISER_METADATA}}
-          payload = {}
-          payload["analysis_id"] = "{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          metadata = json.loads('{{inputs.parameters.THOTH_ADVISER_METADATA}}')
```

Comment:
It might be a good idea to move this sources out of the template and provide a Python s2i app that does this job. The source can grow over time (error handling, ...).

Reply:
Yes agree :) I'll create a card! :)